src/nvme/ioctl.h

```diff
@@ -1137,17 +1137,21 @@ static inline int nvme_zns_identify_ctrl(int fd, struct nvme_zns_id_ctrl *id)
 struct nvme_get_log_args {
 	int args_size;
 	int fd;
+	__u32 timeout;
+	__u32 rsvd12;
```

Comment:
Argl. No. Why add a reserved field into a perfectly aligned structure?

Reply:
Again the idea is to have match with the common members. So it's for the sake of consistency.